src/main/java/com/deavensoft/paketomat/center/PackageServiceImpl.java

```diff
@@ -19,57 +19,53 @@
 
 @Slf4j
 @Service
-public class CenterServiceImpl implements CenterService {
-    private final CenterRepository centerRepository;
+@AllArgsConstructor
+public class PackageServiceImpl implements PackageService {
+    private final PackageRepository packageRepository;
     private final EmailService emailService;
     @Autowired
```

Comment:
Remove the annotation

Reply:
I removed this annotation and put RequiredArgs

Reply:
Check it again

Reply:
Fixed